README.md

```diff
@@ -1 +1,95 @@
-# sumologic-opentelemetry-python
\ No newline at end of file
+# Sumo Logic OpenTelemetry Python
+
+
+An all-in-one package for python projects used to enable OpenTelemetry auto-instrumentation.
+
+It contains all supported propagators and auto-instrumentation plugins.
+
+## Instalation
+
+```
+pip install sumologic-opentelemetry
+```
+
+
+## Instrumented packages
+
+This package installs all officially supported auto-instrumentation packages() as well as comonly used propagators and exporters:
+
+### Instrumented packages
+```
+opentelemetry-instrumentation-aws-lambda
+opentelemetry-instrumentation-dbapi
+opentelemetry-instrumentation-logging
+opentelemetry-instrumentation-sqlite3
+opentelemetry-instrumentation-urllib
+opentelemetry-instrumentation-wsgi
+opentelemetry-instrumentation-aiohttp-client
+opentelemetry-instrumentation-aiopg
+opentelemetry-instrumentation-asgi
+opentelemetry-instrumentation-asyncpg
+opentelemetry-instrumentation-boto
+opentelemetry-instrumentation-botocore
+opentelemetry-instrumentation-celery
+opentelemetry-instrumentation-django
+opentelemetry-instrumentation-elasticsearch
+opentelemetry-instrumentation-falcon
+opentelemetry-instrumentation-fastapi
+opentelemetry-instrumentation-flask
+opentelemetry-instrumentation-grpc
+opentelemetry-instrumentation-httpx
+opentelemetry-instrumentation-jinja2
+opentelemetry-instrumentation-kafka-python
+opentelemetry-instrumentation-mysql
+opentelemetry-instrumentation-pika
+opentelemetry-instrumentation-psycopg2
+opentelemetry-instrumentation-pymemcache
+opentelemetry-instrumentation-pymongo
+opentelemetry-instrumentation-pymysql
+opentelemetry-instrumentation-pyramid
+opentelemetry-instrumentation-redis
+opentelemetry-instrumentation-requests
+opentelemetry-instrumentation-sklearn
+opentelemetry-instrumentation-sqlalchemy
+opentelemetry-instrumentation-starlette
+opentelemetry-instrumentation-tornado
+opentelemetry-instrumentation-urllib3
+```
```

Comment:
I would make it list with links to code/documentation

Reply:
Is redirecting to pypi packages enough? 
Most packages have pointer to the documentation, i.e. https://pypi.org/project/opentelemetry-instrumentation-urllib3/

Reply:
Yes, pointing to these docs should be fine

Reply:
Sorry, I though You were meaning to link every module separately e.g. `[opentelemetry-instrumentation-urllib3](https://pypi.org/project/opentelemetry-instrumentation-urllib3/)`

Reply:
It is done separatly.